hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -276,26 +277,26 @@
     /**
      * Controls the frequency of a BackupAwareOperation getting its async backups converted to a sync backups. This is needed
      * to prevent an accumulation of asynchronous backups and eventually running into stability issues.
-     *
+     * <p/>
```

Comment:
Please remove all these <p/>. They are illegal for Java 8. So please update your intellij to stop inserting it.